src/niivue/index.ts

```diff
@@ -104,6 +104,7 @@ export * from '../nvdocument.js'
 export { NVUtilities } from '../nvutilities.js'
 export { LabelTextAlignment, LabelLineTerminator, NVLabel3DStyle, NVLabel3D } from '../nvlabel.js'
 export { NVMeshLoaders } from '../nvmesh-loaders.js'
+export { Connectome, LegacyConnectome} from "../types.js"
```

Comment:
@hanayik adding this line causes `npm run dev` to fail for me. I think it should be removed from this PR.

Reply:
The line just needs to be modified, rather than removed. I will submit a new commit once local tests pass.